src/components/forms/AddRecipientForm.js

```diff
@@ -87,13 +98,30 @@ function AddRecipientForm({ visible, onCreate, onCancel, households }) {
             ]}
           >
             <Select placeholder="Please select their gender">
-              <Option value="male">Male</Option>
-              <Option value="female">Female</Option>
-              <Option value="nonbinary">Non-Binary Gender</Option>
+              <Option value={1}>Female</Option>
```

Comment:
Same as comments in previous file. These values should be referenced from the DB instead of hardcoded. Please document for future PR.